keyboards/wilba_tech/wt_rgb_backlight.c

```diff
@@ -1044,6 +1045,35 @@ const Point g_map_led_to_point_polar[BACKLIGHT_LED_COUNT] PROGMEM = {
     {0,9}, {0,43}, {0,77}, {0,111}, {0,145}, {0,201}, {224,181}, {230,217}, {235,255},
     {189,128}, {200,131}, {210,141}, {218,159}, {255,255}, {207,238}, {211,255}, {218,255}, {221,255}
 };
+#elif defined(RGB_BACKLIGHT_WT65_C) || defined(RGB_BACKLIGHT_WT65_CX)
```

Comment:
Should be using core RGB matrix feature.

Reply:
Core RGB matrix does not have the same features. I'll refactor when it does.

Reply:
What features is it missing? 

Because if it's turning on/off lights based on the layout selection in via, that this absolutely can be done with the core rgb matrix feature.  In fact, the work louder work board is doing exactly that. 

Otherwise, specifically, what functionality is missing?

Reply:
Storing two colors, so the effects can use two colors, something myself and others have suggested for years.

The point is, this is the firmware that shipped on the PCBs. It can be stored here, among the other wilba.tech firmware. Then we can have a discussion about the best way to refactor the WT RGB code to maintain the same functionality. I am now more motivated to refactor after seeing the recent changes to Core RGB Matrix, but I still think it would be nice if the refactor would not involve duplicating effects just so they use two colors.

Reply:
There are only 3 modes that use dual colors, from what the code shows. 
* Alpha Mods
* Gradient Up-Down
* Raindrops

Alpha Mods and Gradient explicitly use the "speed" setting to set the secondary color in the qmk core feature, as neither are active animations (eg, they're static, so no "speed" is needed). 

As for Raindrops, That one is a bit more muddled, and isn't a 1:1 translation. 

But staying on custom code for one animation is ridiculous.  Especially since ...

Part of my concern here is that the wt backlight code is not flexible, at all when it comes to user customization, and is an issue that has come up for users repeatedly.

Reply:
also, will see about opening an issue to continue the discussion, as to not interfere with this PR